ui/page/account/account_page.go

```diff
@@ -0,0 +1,273 @@
+				return layout.Flex{Axis: layout.Vertical}.Layout(gtx,
+					layout.Rigid(pg.headerLayout),
+					layout.Rigid(func(gtx C) D {
+						return pg.accountsList.Layout(gtx, len(pg.accounts), func(gtx C, i int) D {
+							return pg.accountItem(gtx, pg.accounts[i], i == len(pg.accounts)-1)
+						})
+					}),
+				)
+			})
+		})
+	})
+}
+
+func (pg *Page) headerLayout(gtx C) D {
+	return layout.Inset{Bottom: values.MarginPaddingMinus12}.Layout(gtx, func(gtx C) D {
+		return layout.Flex{Spacing: layout.SpaceBetween, Alignment: layout.Middle}.Layout(gtx,
+			layout.Rigid(func(gtx C) D {
+				txt := pg.Theme.Label(values.TextSize20, values.String(values.StrAccounts))
+				txt.Font.Weight = font.SemiBold
+				return txt.Layout(gtx)
+			}),
+			layout.Flexed(1, func(gtx C) D {
+				return layout.E.Layout(gtx, pg.addAccountLayout)
+			}),
+		)
+	})
+}
+
+func (pg *Page) addAccountLayout(gtx C) D {
+	return layout.Inset{
+		Left:   values.MarginPadding24,
+		Top:    values.MarginPadding16,
+		Bottom: values.MarginPadding24,
+	}.Layout(gtx, func(gtx C) D {
+		return cryptomaterial.LinearLayout{
+			Width:      cryptomaterial.WrapContent,
+			Height:     cryptomaterial.WrapContent,
+			Background: pg.Theme.Color.DefaultThemeColors().SurfaceHighlight,
+			Clickable:  pg.addAccountBtn,
+			Alignment:  layout.Middle,
+		}.Layout(gtx,
+			layout.Rigid(pg.Theme.Icons.BlueAddIcon.Layout16dp),
+			layout.Rigid(func(gtx C) D {
+				txt := pg.Theme.Label(values.TextSize16, values.String(values.StrAddNewAccount))
+				txt.Color = pg.Theme.Color.DefaultThemeColors().Primary
+				txt.Font.Weight = font.SemiBold
+				return layout.Inset{
+					Left: values.MarginPadding8,
+				}.Layout(gtx, txt.Layout)
+			}),
+		)
+	})
+}
+
+func (pg *Page) accountItem(gtx C, account *sharedW.Account, isHiddenLine bool) D {
+	return layout.Flex{Axis: layout.Vertical}.Layout(gtx,
+		layout.Rigid(func(gtx C) D {
+			return layout.Inset{Top: values.MarginPadding36}.Layout(gtx, func(gtx C) D {
+				return layout.Flex{}.Layout(gtx,
+					layout.Rigid(func(gtx C) D {
+						txt := pg.Theme.Label(values.TextSize18, account.AccountName)
+						txt.Font.Weight = font.SemiBold
+						return txt.Layout(gtx)
+					}),
+					layout.Flexed(1, func(gtx C) D {
+						balance := account.Balance.Total.String()
+						totalBalance := pg.wallet.ToAmount(account.Balance.Total.ToInt()).ToCoin()
+						if pg.exchangeRate != -1 && pg.usdExchangeSet {
+							balanceUSD := utils.FormatAsUSDString(pg.Printer, utils.CryptoToUSD(pg.exchangeRate, totalBalance))
+							balance = fmt.Sprintf("%s (%s)", balance, balanceUSD)
+						}
+						txt := pg.Theme.Label(values.TextSize18, balance)
+						txt.Font.Weight = font.SemiBold
+						return layout.E.Layout(gtx, txt.Layout)
+					}),
+				)
+			})
+		}),
+		layout.Rigid(func(gtx C) D {
+			return layout.Inset{Top: values.MarginPadding16, Bottom: values.MarginPadding28}.Layout(gtx, func(gtx C) D {
```

Comment:
```suggestion
			return layout.Inset{Top: values.MarginPadding16, Bottom: values.MarginPadding24}.Layout(gtx, func(gtx C) D {
```

Reply:
@dreacot  it is 28 not 24

![image](https://github.com/crypto-power/cryptopower/assets/19331824/eff10141-8613-41c9-af26-5eaf1ad5c02e)